config/config.go

```diff
@@ -496,6 +534,85 @@ func (c *ScrapeConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+// AlertingConfig configures alerting and alertmanager related configs
+type AlertingConfig struct {
+	AlertRelabelConfigs  []*RelabelConfig       `yaml:"alert_relabel_configs,omitempty"`
+	AlertmanagersConfigs []*AlertmanagersConfig `yaml:"alertmanagers,omitempty"`
```

Comment:
Calling this `AlertmanagersConfigs` seems like it's going to cause confusion, as previously where we had a list the name of the items was singular (e.g. scrape_config). I can't think of a better name though.